### ISSUE TITLE
Is it possible to assing "Win + Shift + S" to capture region?

### ISSUE THREAD

**Issue by theofbonin** [ShareX/ShareX#7052]
I am used to using it because of snipping tool, but I tried uninstalling snipping tool to use the hotkey on sharex and it didn't work. I tried disabling the S shortcut on regedit and it works but also disable "Win + S" which I always use. Couldn't find any other way to use it on sharex and disable it on windows/snipping tool.

**Comment by accountnujen** [comment]
UP! I have the default screenshot removed, however I can't set the hotkey for your app: win +shift + s.

**Comment by Payne530** [comment]
I don't think this is an issue with ShareX, but more of an issue with Windows. I am also used to using the "Windows + Shift + S" keybind for doing a snapshot (works with one hand). After uninstalling both snipping tool and snip & sketch applications, I can still press the combo and the tool still shows up.

It seems that Windows does not allow the user to remove the keybind for the "Windows + Shift + S" command.

**Comment by Elon61** [comment]
While it's true that windows doesn't really allow you to disable that, ShareX should be able to catch the keybind before windows. Utilities such as [Autohotkey](https://www.autohotkey.com/docs/v1/misc/Override.htm) do just that, which allows you to remap win+shift+s to whatever you want. It would be pretty nice to have that built into ShareX.

**Comment by Dott-rus** [comment]
up! i really need this!

**Comment by arthur-er** [comment]
You can do it via regedit, here's the code for the disable, you can then assign win+shift+s to sharex

```
Windows Registry Editor Version 5.00

[HKEY_CURRENT_USER\Software\Microsoft\Windows\CurrentVersion\Explorer\Advanced]
"DisabledHotkeys"="S"
```

**Comment by towlenner** [comment]
If you have PowerToys then the [Keyboard Manager](https://learn.microsoft.com/en-gb/windows/powertoys/keyboard-manager) can be used to map Win+Shift+S to Ctrl+Prt Scr

**Comment by jonnywright** [comment]
> If you have PowerToys then the [Keyboard Manager](https://learn.microsoft.com/en-gb/windows/powertoys/keyboard-manager) can be used to map Win+Shift+S to Ctrl+Prt Scr

I've tried this but it doesn't seem to intercept it - Snipping Tool still takes precedence.

**Comment by superjose** [comment]
I was able to do it following @arthur-er 's recommendation. I'm going to expand on his answer:

1st. Open the registry:
(Right Click on Start -> Run -> Type: `regedit`) 

2nd. Navigate to:
```
Computer\HKEY_CURRENT_USER\Software\Microsoft\Windows\CurrentVersion\Explorer\Advanced
```
3rd. Right-click on an empty space -> New -> String Value 
4th. Name: `DisabledHotkeys` ; Value:  `S`
5th. Restart Explorer:
* Mouse on your taskbar -> `Hold Both: Ctrl + Shift` and `right click` -> Exit Explorer
* Press `Ctrl + Shift + Esc` (Opens up Task Manager)
* New Task -> `explorer`

This disables it entirely. 

If you want your original behavior, delete the `DisabledHotkeys` and restart explorer like in step 5.

**Comment by b1k3rdude** [comment]
Following @arthur-er  advice, like @superjose I can also confirm this reg hack works.

**Comment by brycekunkel** [comment]
I tried Power Toys Keyboard Manager this morning and the remap to Ctrl + PntScn worked and overrides the default snipping tool (in Windows 10 - this may change when my company finally pushes out Windows 11).

**Comment by mystyle-48** [comment]
If you wish to still use the Windows keybinds that rely on `R` and `S` (`Win`+`R` Run menu, `Win`+`S` search), do not use `DisabledHotkeys`. Use this machine-wide group policy instead:

```
Windows Registry Editor Version 5.00

[HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\PolicyManager\default\Experience\AllowScreenCapture]
"value"=dword:00000000
```

As suggested above, the PowerToys Keyboard Manager may then be used to map `Win`+`Shift`+`S` to the ShareX capture region hotkey.

**Comment by cachius** [comment]
> \> If you have PowerToys then the [Keyboard Manager](https://learn.microsoft.com/en-gb/windows/powertoys/keyboard-manager) can be used to map Win+Shift+S to Ctrl+Prt Scr
> 
> @jonnywright I've tried this but it doesn't seem to intercept it - Snipping Tool still takes precedence.

Works ~flawlessly~ in programs running as that user. **Programs which are running as _another_ user like Administrator still trigger Snipping Tool**. If you never do this, you can use this solution.
Screenshot from Keyboard Manager for reference (in German):

<img width="670" height="255" alt="Image" src="https://github.com/user-attachments/assets/a30173c1-3a61-4c26-8df5-6022e0e934ec" />

---

Setting `DisabledHotkeys` in `HKCU\Software\Microsoft\Windows\CurrentVersion\Explorer\Advanced` works, and for _all_ windows regardless of user context. Can be set with one command:
```
reg.exe add "HKCU\Software\Microsoft\Windows\CurrentVersion\Explorer\Advanced" /v "DisabledHotkeys" /t REG_SZ /d S
```
Then you can set the shortcut in ShareX directly (notice how it orders Shift before Win key, opposite to [Snipping Tool](https://support.microsoft.com/en-us/windows/use-snipping-tool-to-capture-screenshots-00246869-1843-655f-f220-97299b865f6b) and PowerToys):

<img width="609" height="102" alt="Image" src="https://github.com/user-attachments/assets/de1a3d80-ba09-451b-bd11-4ab45d9fb7a5" />

**Comment by cachius** [comment]
Addendum: Once Win + Shift + S for Snipping tool was disabled via Explorer\Advanced "DisabledHotkeys"="S", after a reboot [OneNotes Win +S shortcut](https://superuser.com/questions/199422/how-to-disable-or-change-the-windowss-hotkey-in-onenote) interfered with the ShareX setting. I got 'Unable to set shortcut' message from ShareX until I hid OneNote from the toolbar. It wasn't a problem at all times, though.
```
ShareX - Hotkey registration failed

Unable to register hotkey:
[Shift + Win + S] Capture region
Please select a different hotkey or quit the conflicting application and reopen ShareX.
```
<img width="466" height="247" alt="Image" src="https://github.com/user-attachments/assets/e90f4aa3-c317-4292-ae5f-a4b6f7582335" />
